app/models/results/web_response_parser.rb

```diff
@@ -66,7 +66,7 @@ def update_or_add_node(web_hash_node, tree_parent)
         tree_parent.children.build(new_tree_node_attrs(web_hash_node, tree_parent))
       else # update
         existing_node = tree_parent.children.select { |c| c.id == id }.first
-        updatable_params = web_hash_node.slice(:value).permit(PERMITTED_PARAMS)
```

Comment:
@DevneyHamilton @smoyth I had to remove this in order to get some things working.  Is it okay to make all permitted params updatable?

Reply:
Hard to see the way this comment is being displayed but the line changed to:

```ruby
updatable_params = web_hash_node.permit(PERMITTED_PARAMS)
```